i18n/about/code-of-conduct.i18n.js

```diff
@@ -5,51 +5,53 @@ export default genI18nMessages({
         title: 'Code of Conduct',
         pageAbstract:
             'PyCon Taiwan aims to provide a joyous, pleasant and vibrant environment. ' +
-            'This convention applies to all the words and deeds of participants in the' +
-            'relevant entities and digital fields of PyCon Taiwan. It is hoped that everyone will' +
-            'work together to make PyCon Taiwan full of mutual respect and trust.',
+            'This convention applies to all the words and deeds of participants in ' +
+            'therelevant entities and digital fields of PyCon Taiwan. It is hoped ' +
+            'that everyone willwork together to make PyCon Taiwan full of mutual respect and trust.',
```

Comment:
**issue (blocking):** There are some terms are concatenated. Please help fix that, thanks!

Reply:
OK